h2o-py/tests/testdir_misc/pyunit_metric_accessors.py

```diff
@@ -108,6 +113,16 @@ def metric_accessors():
     gbm = H2OGradientBoostingEstimator(nfolds=3, distribution=distribution, fold_assignment="Random")
     gbm.train(y=response_col, x=predictors, validation_frame=valid, training_frame=train)
 
+    # using list from http://docs.h2o.ai/h2o/latest-stable/h2o-docs/performance-and-prediction.html#classification
+    # + common ones
+    for metric in ['gini', 'logloss', 'auc', 'aucpr', 'mse', 'rmse']:
+        val = getattr(gbm, metric)()
+        assert isinstance(val, float), "expected a float for metric {} but got {}".format(metric, val)
+
+    for metric in ['mcc', 'F1', 'F0point5', 'F2', 'accuracy', 'mean_per_class_error']:
+        val = getattr(gbm, metric)()[0][1]
+        assert isinstance(val, float), "expected a float for metric {} but got {}".format(metric, val)
+
```

Comment:
please admire the consistency of our metrics accessors...
Please +1 if I should create a JIRA for this, knowing that it will be difficult to make this backwards compatible.